guake/gsettings.py

```diff
@@ -320,7 +320,7 @@ def fstyle_changed(self, settings, key, user_data):
 
         if terminal_uuid:
             terminal = self.guake.notebook_manager.get_terminal_by_uuid(terminal_uuid)
-            terminals = (terminal,) if terminal else tuple()
+            terminals = (terminal,) if terminal else ()
```

Comment:
i think this this quite wrong actually... either we use a list or a `(None,)` but having `terminals` sometimes 1-tuple, sometimes 0-tuple (why does this concept even exist?? it makes no sense).

so I vote to turn `terminals` into list.